src/ui/public/agg_types/controls/interval.html

```diff
@@ -1,4 +1,16 @@
 <div class="form-group">
+
+  <div
+    class="vis-option-item indented"
+    ng-show="!agg.fieldIsTimeField()">
+    <label>
+      <input type="checkbox"
+             name="isFilteredByKbnTime"
+             ng-model="agg.params.isFilteredByKbnTime">
+      Only request data within Kibana time period
```

Comment:
I think `Ignore the global time range` is very misleading because it can never be ignored. The ES request will always have a query against the global time range.

I tried `Restrict to global time range` which I think does a better job describing the effect of the variable